integration/aggregate_documents_compat_test.go

```diff
@@ -39,6 +39,7 @@ type aggregateStagesCompatTestCase struct {
 	resultType     compatTestCaseResultType // defaults to nonEmptyResult
 	resultPushdown bool                     // defaults to false
 	skip           string                   // skip test for all handlers, must have issue number mentioned
+	failsForSQLite string
```

Comment:
I expected if you leave `failsForSQLite` empty for a test that should fail, it should fail. But that doesn't seem to be the case. Could you add comment how this behaves please :hugs: 

I think if a test that passes has non-empty `failsForSQLite`, it should fail.
If a test that fails has empty `failsForSQLite`, it should fail.
Other tests should pass.